src/gridfs/file.rs

```diff
@@ -201,17 +201,17 @@ impl File {
                 try!(self.gfs.files.insert_one(self.doc.to_bson(), None));
 
                 // Ensure indexes
-                try!(self.gfs.files.create_index(doc!{ "filename" => 1 }, None));
+                try!(self.gfs.files.create_index(doc!{ "filename": 1 }, None));
 
                 let mut opts = IndexOptions::new();
                 opts.unique = Some(true);
                 try!(self.gfs.chunks.create_index(
-                    doc!{ "files_id" => 1, "n" => 1},
+                    doc!{ "files_id": 1, "n": 1},
```

Comment:
Ditto above about multiple keys

It's not super egregious here since they're small, but I think that it will never look worse to split it, and I think consistency is better than just winging it